app/src/main/java/com/eleks/mowid/ui/feature/home/HomeScreen.kt

```diff
@@ -68,10 +68,14 @@ fun HomeScreen(viewModel: HomeViewModel) {
         viewModel.event.onEach { event ->
             when (event) {
                 HomeEvent.ShowAddGroupModal -> {
-                    if (bottomSheetScaffoldState.bottomSheetState.isCollapsed) {
-                        bottomSheetScaffoldState.bottomSheetState.expand()
+                    if (activityViewModel.isUserAlreadyLogIn()) {
+                        if (bottomSheetScaffoldState.bottomSheetState.isCollapsed) {
+                            bottomSheetScaffoldState.bottomSheetState.expand()
+                        } else {
+                            bottomSheetScaffoldState.bottomSheetState.collapse()
```

Comment:
Sorry, didn't get why do you collapse bottom sheet when it is not Collapsed and when the user is already logged in. Could you please elaborate more on this? Thanks!